negotiation/neural/model_builder.py

```diff
@@ -31,8 +31,8 @@ def add_model_arguments(parser):
                        help="""Type of encoder layer to use. Non-RNN layers
                        are experimental. Options are
                        [rnn|brnn|mean|transformer|cnn].""")
-    group.add_argument('--decoder-type', type=str, default='rnn',
-                       choices=['rnn', 'transformer', 'cnn'],
+    group.add_argument('--decoder-type', type=str, default='multibank',
+                       choices=['rnn', 'transformer', 'cnn', 'multibank'],
```

Comment:
the multibank should be used to specify attention type. the decoder is still rnn. can you check opt.global_attention? can we specify different types of attention there?